frontend/src/App.js

```diff
@@ -22,13 +22,17 @@ import UpdateCategory from './component/admin/category/updateCategory/updateCate
 import Sidebar from './component/sidebar/sidebar'
 import ProductView from './component/seller/product/productView'
 
+import Dashboard from './component/admin/dashboard/dashboard'
+import Profile from './component/admin/profile/profile';
+import ViewRole from './component/admin/role/viewRole/viewRole';
+
 
 function App() {
   return (
     <div className="App">
       <BrowserRouter>
-        <NavBar />
-        <Home/>
+        {/* <NavBar />
+        <Home/> */}
```

Comment:
Home component have to be inside routes. Like this
<Route path="/home" element={<Home />} exact></Route>